pyvo/registry/rtcons.py

```diff
@@ -719,9 +719,16 @@ def tomoc(s):
 
         elif len(geom_spec) == 2:
             if isinstance(geom_spec[0], SkyCoord):
+                # If radius given is astropy quantity, then convert to degrees
+                if isinstance(geom_spec[1], u.Quantity):
+                    if geom_spec[1].unit.physical_type != 'angle':
+                        raise ValueError("Radius quantity is not of type dimensionless angle.")
```

Comment:
Please update this error message, there are non-dimensionless angle units that work here, too.

Reply:
Whoops, thanks for that! Fixed in https://github.com/astropy/pyvo/pull/594/commits/c0c135746021d35a2fb7ec16603f2972c00caa59